src/dbus.c

```diff
@@ -1202,7 +1202,7 @@ static bool cdbus_process_opts_get(session_t *ps, DBusMessage *msg) {
 	cdbus_m_opts_get_do(shadow_offset_x, cdbus_reply_int32);
 	cdbus_m_opts_get_do(shadow_offset_y, cdbus_reply_int32);
 	cdbus_m_opts_get_do(shadow_opacity, cdbus_reply_double);
-	cdbus_m_opts_get_do(xinerama_shadow_crop, cdbus_reply_bool);
+	cdbus_m_opts_get_do(crop_shadow_to_monitor, cdbus_reply_bool);
```

Comment:
@yshui Do we want a fallback to the old "deprecated" value here as well? Or just continue to treat the dbus interface as unstable?

Reply:
hmm, i haven't thought about this. as i can see, the dbus interface currently accepts deprecated options without any notification about them being deprecated, so it could be
```c
cdbus_m_opts_get_do(xinerama_shadow_crop, cdbus_reply_bool);
cdbus_m_opts_get_do(crop_shadow_to_monitor, cdbus_reply_bool);
```
instead with the first one removed when the deprecation process ends.

Reply:
the dbus interface is still undocumented, there is still no way to enumerate properties and methods. we need to fix these, but before we do i think we can consider it unstable.

Reply:
@yshui, so no fallback to the old option name here?